README.md

```diff
@@ -6,7 +6,7 @@ GraphQLCRUD Java automates schema generation and integrates with GraphQL CRUD ca
 
 You can run your application in dev mode that enables live coding using:
 ```
-./mvnw quarkus:dev
+mvn clean install
```

Comment:
Good catch! Also noticed other quarkus related bits in the [pom.xml](https://github.com/graphqlcrud/graphqlcrud-java/blob/master/pom.xml), they can be cleaned later on.